tests/test_diaForcedSource.py

```diff
@@ -67,15 +66,6 @@ def create_test_dia_objects(n_points, wcs, startPos=100):
 class TestDiaForcedSource(unittest.TestCase):
 
     def setUp(self):
-        # CFHT Filters from the camera mapper.
-        self.filter_names = ["u", "g", "r", "i", "z"]
```

Comment:
Don't you still need `self.filter_names`?

Reply:
It appears it was never used in this particular test. The only test that still needs it is `test_association_task.py`